### ISSUE TITLE
Voxel Probleme 

### ISSUE THREAD

**Issue by wither735** [carl-vbn/minecraft-voxel-loader#1]
Hello its me Wither from youre You Tube Video xD I have a probleme when i try to voxel a model i always get this probleme i think its because i use the blender file browser to select a directory. I hope you can help me sorry that i am so annoying xD
![2021-08-21](https://user-images.githubusercontent.com/89296018/130316193-bc76d182-49ff-45b9-a879-f518e6f2f77f.png)

**Comment by carl-vbn** [comment]
The problem doesn't seem to come from the script itself but it looks like you either did not save the file or save it at a location where Blender doesn't have permission to write. Please try to save the blender file on your Desktop or in another directory where it has write permissions (Documents should also work) and try again. I'm NOT talking about the "Ouput directory" field of the panel but the place where you saved the .blend file using File > Save. 

I also see that your frame range is set to start and end at 0, but your animation goes from frame 1 to frame 50. Please change the frame range in the Voxelizer panel to this range as well.

**Comment by wither735** [comment]
Thanks for your help, I have now tried it and now I get another short error message. Guilty if my english is not that good I am from Germany
![2021-08-21 (1)](https://user-images.githubusercontent.com/89296018/130323136-346e96ff-2590-429f-b2d1-0042d7f7a6c2.png)

**Comment by carl-vbn** [comment]
This error message is because your output directory `//.\Desktop\` is relative, which the voxelizing script does not support. Instead it should be absolute (starting with a drive letter like `C:\`). If you want to export the frames to the Desktop, it should be `C:\Users\angel\Desktop\` (I am guessing `angel` is your username from the previous screenshot). 

Keep in mind that the extension will create one file per frame, so I would recommend creating a folder on your Desktop and exporting your frames there, so the output dir would be `C:\Users\angel\Desktop\NAME_OF_THE_FOLDER\`.

This should then work without any issue.

**Comment by wither735** [comment]
Thanks it really worked ok i will do the next steps now. You are so nice :D

**Comment by wither735** [comment]
I have one qestion @carl-vbn now i need to download the minecraft mod is it the grandle wrapper?

**Comment by carl-vbn** [comment]
I'm glad the script worked for you! The mod file can be downloaded directly from the [releases](https://github.com/carl-vbn/minecraft-voxel-loader/releases/). Here's a [direct link to download the .jar file](https://github.com/carl-vbn/minecraft-voxel-loader/releases/download/1.0/vxloader-mc1.17-v1.0.jar) (your browser will probably flag it as dangerous, but that's just because it hasn't been downloaded enough times to be considered trustworthy). If you want to build the mod yourself, download the entire repository and run `gradlew build` in a terminal inside the directory.
Keep in mind that the mod currently only works for Minecraft 1.17 (not 1.17.1 or later).

**Comment by wither735** [comment]
ok thank you ;D

**Comment by carl-vbn** [comment]
No problem. Please let me know if everything works out for you, so I can close this issue.

**Comment by wither735** [comment]
Ok i am Almost finnish i only have a little probleme with the commands so i know that i first have to load the animation befor i can play it but what i have to write in the dirname. I already put all the blocks like ( 0.blocks ) in the voxelData folder
![2021-08-21 (6)](https://user-images.githubusercontent.com/89296018/130329147-9ccef3f0-fc19-4eff-8417-49b9613e8f95.png)

**Comment by carl-vbn** [comment]
You have to create another folder inside voxelData and put all the .blocks files in there. `dirname` is the name of that folder.

**Comment by wither735** [comment]
Ohh thx i will try it :D

**Comment by wither735** [comment]
Thank you i am finnaly finnish here is the Video in Blender and in Minecraft (just a test animation)
![2021-08-21 (1)](https://user-images.githubusercontent.com/89296018/130331776-d5939da4-6dbf-407c-8252-5a0b19383bbe.png)

https://user-images.githubusercontent.com/89296018/130331783-25ae0f39-adf8-41ae-9da2-d13d488f6532.mp4

**Comment by wither735** [comment]
I will now try it on a normel animation and i will try to fix the probleme with the texture :D 
And is there any link or something where i can donate because you are such a great person and you help me in Youre free time

**Comment by carl-vbn** [comment]
I'm glad to see that you successfully managed to play an animation inside the game! If you don't want any gaps between the blocks, use a subsurface modifier in blender set to "simple" mode. And what is your problem with the texture exactly? 
It's also very kind of you to consider donating, but I don't have such a system in place at the moment. I'm always happy to help! If you really want to support me, sharing my channel with people who might be interested is the best way!

**Comment by wither735** [comment]
Hello again i tried something similar like you but it dont looks so well is there any way to make it bigger so that it use more blocks so that it looks more like a ball.

https://user-images.githubusercontent.com/89296018/130346849-3e816329-6919-44dd-9e53-3bc4c1ad0931.mp4


https://user-images.githubusercontent.com/89296018/130346851-954033ef-3ff4-4f69-9419-24df62432a08.mp4

**Comment by carl-vbn** [comment]
You have to scale the sphere up in Blender. One meter is one block.

**Comment by wither735** [comment]
Oh thank i am stupid i already read that in the info xD

**Comment by wither735** [comment]
Oh and by the way you reachd 2k 🎉🎊

**Comment by carl-vbn** [comment]
Yes I know it's incredible how fast the channel is growing recently!!

**Comment by wither735** [comment]
Hello again i now tried a (normel) animation but first i had some issues with the textures but then i fixd it and here is the final animation its not as good as yours xD (I was too lazy to animate the legs) and sorry that all the Videos that i send you had such a bad quality its because i can only send you Videos that are under 10 Mb 
![2021-08-22 (2)](https://user-images.githubusercontent.com/89296018/130353760-942dc31d-e479-43db-8618-c0098b6bac7e.png)

https://user-images.githubusercontent.com/89296018/130353851-7b491175-ec97-4c95-9a74-05f817500151.mp4

**Comment by carl-vbn** [comment]
That looks great, good job!
